code/game/area/ss13_areas.dm

```diff
@@ -1532,6 +1532,10 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 /area/security/vacantoffice2
 	name = "\improper Vacant Office"
 	icon_state = "security"
+	
+/area/security/permasolitary
+	name = "\improper Solitary Confinement"
```

Comment:
I'm pretty sure this is legacy code and that it's not needed. Someone can correct me if I'm wrong.
```suggestion
	name = "Solitary Confinement"
```

Reply:
Since I'm relatively new to mapping I styled it this way to copy most of the other areas. I'll commit this and if it causes issues it's easy enough to revert.